### PR TITLE
cloud: DM supports physical mode

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

As tittle

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 1]
```diff
@@ -195,10 +205,33 @@ In the **Choose the objects to be migrated** step, you can choose existing data
 
 To migrate data to TiDB Cloud once and for all, choose both **Existing data migration** and **Incremental data migration**, which ensures data consistency between the source and target databases.
 
+You can use [physical mode](https://docs.pingcap.com/tidb/stable/tidb-lightning-physical-import-mode) or [logical mode](https://docs.pingcap.com/tidb/stable/tidb-lightning-logical-import-mode) to migrate existing data.
+
+- The default mode is **Logical mode**. This mode supports migrating data into a table with existing data. But the performance is slower than physical mode.
+
+- If you use **Physical mode**, the migration job will use [TiDB Lightning](https://docs.pingcap.com/tidb/stable/tidb-lightning-overview) to migrate the existing data. This mode is recommended for large datasets. When you use this mode, the target table must be empty. The performance is about 2.5 times faster than the logical mode.
```
Comment (Frank945946) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:212]: 就不要暴露 Lightning 了，减少用户理解复杂度

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 2]
```diff
@@ -195,10 +205,33 @@ In the **Choose the objects to be migrated** step, you can choose existing data
...
+- The default mode is **Logical mode**. This mode supports migrating data into a table with existing data. But the performance is slower than physical mode.
+
+- If you use **Physical mode**, the migration job will use [TiDB Lightning](https://docs.pingcap.com/tidb/stable/tidb-lightning-overview) to migrate the existing data. This mode is recommended for large datasets. When you use this mode, the target table must be empty. The performance is about 2.5 times faster than the logical mode. 
+
+There are limitations for using physical mode:
+
+- When you use physical mode, you cannot create a second migration job before the existing data migration is completed.
+- Physical mode is available only for TiDB clusters in AWS regions.
+
+Physical mode exports the upstream data as fast as possible, so [different specifications](/tidb-cloud/tidb-cloud-billing-dm.md#specifications-for-data-migration) have different performance impacts on QPS and TPS of the upstream database during data export. The following table shows the performance regression of each specification.
```
Comment (Frank945946) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:219]: 加粗强调一下是上游数据库，或者在下面的表格第二列表示是上游数据库的性能衰退

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 3]
```diff
@@ -195,10 +205,33 @@ In the **Choose the objects to be migrated** step, you can choose existing data
...
+
+| Migration specifications | Performance regression | Export speed (-t=2*rcu) |
```
Comment (Frank945946) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:221]:
```suggestion
| Migration specifications | Performance regression | Maximum Export speed |
```

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 4]
```diff
@@ -195,10 +205,33 @@ In the **Choose the objects to be migrated** step, you can choose existing data
...
+|-------|-------|----------|
+| Without DM    | /   |  / |
```
Comment (Frank945946) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:223]:
```suggestion

```

File: tidb-cloud/migrate-from-mysql-using-data-migration.md (hunk 1 above)
Comment (hfxsd) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:212]:
```suggestion
- **Physical mode** is recommended for large datasets. When you use this mode, the target table must be empty. The performance is about 2.5 times faster than the logical mode. 
```

File: tidb-cloud/migrate-from-mysql-using-data-migration.md (hunk 2 above)
Comment (hfxsd) [reply tidb-cloud/migrate-from-mysql-using-data-migration.md:219]: Done

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 5]
```diff
@@ -195,10 +205,33 @@ In the **Choose the objects to be migrated** step, you can choose existing data
 
 To migrate data to TiDB Cloud once and for all, choose both **Existing data migration** and **Incremental data migration**, which ensures data consistency between the source and target databases.
 
+You can use [physical mode](https://docs.pingcap.com/tidb/stable/tidb-lightning-physical-import-mode) or [logical mode](https://docs.pingcap.com/tidb/stable/tidb-lightning-logical-import-mode) to migrate existing data.
+
+- The default mode is **Logical mode**. This mode supports migrating data into a table with existing data. But the performance is slower than physical mode.
+
+- If you use **Physical mode**, the migration job uses [TiDB Lightning](https://docs.pingcap.com/tidb/stable/tidb-lightning-overview) in background to migrate the existing data. This mode is recommended for large datasets. When you use this mode, the target table must be empty. The performance is about 2.5 times faster than logical mode.
```
Comment (hfxsd) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:212]:
```suggestion
- It is recommended to use **Physical mode** for large datasets. When you use this mode, the target table must be empty. The performance is about 2.5 times faster than logical mode.
```

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 6]
```diff
@@ -14,25 +14,35 @@ If you want to migrate incremental data only, see [Migrate Incremental Data from
...
 
+- When you delete a cluster in TiDB Cloud, all migration jobs in that cluster are automatically deleted and not recoverable.
+
+### Limitations of existing data migration
+
 - During existing data migration, if the table to be migrated already exists in the target database with duplicated keys, the duplicate keys will be replaced.
 
-- During incremental data migration, if the table to be migrated already exists in the target database with duplicated keys, an error is reported and the migration is interrupted. In this situation, you need to make sure whether the upstream data is accurate. If yes, click the "Restart" button of the migration job and the migration job will replace the downstream conflicting records with the upstream records.
+- If your dataset size is smaller than 1 TiB, it is recommended that you use logical mode (the default mode). If your dataset size is larger than 1 TiB, you can use physical mode. For more information, see [Migrate existing data and incremental data](#migrate-existing-data-and-incremental-data).
```
Comment (Frank945946) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:39]:
```suggestion
- If your dataset size is smaller than 1 TiB, it is recommended that you use logical mode (the default mode). If your dataset size is larger than 1 TiB, or you want to migrate existing data faster, you can use physical mode. For more information, see [Migrate existing data and incremental data](#migrate-existing-data-and-incremental-data).
```

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 7]
```diff
@@ -195,10 +205,32 @@ In the **Choose the objects to be migrated** step, you can choose existing data
 
 To migrate data to TiDB Cloud once and for all, choose both **Existing data migration** and **Incremental data migration**, which ensures data consistency between the source and target databases.
 
+You can use [physical mode](https://docs.pingcap.com/tidb/stable/tidb-lightning-physical-import-mode) or [logical mode](https://docs.pingcap.com/tidb/stable/tidb-lightning-logical-import-mode) to migrate **existing data**.
+
+- The default mode is **Logical mode**. This mode supports migrating data into a table with existing data. But the performance is slower than physical mode.
+
+- It is recommended to use **Physical mode** for large datasets. When you use this mode, the target table must be empty. For the specification of 16RCU, the performance is about 2.5 times faster than logical mode. The performance of other specifications can also increase by more than 100% compared with logcial mode.
```
Comment (Frank945946) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:212]:
```suggestion
- It is recommended to use **Physical mode** for large datasets. When you use this mode, the target table must be empty. For the specification of 16RCU, the performance is about 2.5 times faster than logical mode. The performance of other specifications can also increase by 20% to 50% compared with logical mode. (The performance data is for reference only and may vary in different scenarios.)
```

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 8]
```diff
@@ -195,10 +205,32 @@ In the **Choose the objects to be migrated** step, you can choose existing data
...
+
+There are limitations for using physical mode:
+
+- When you use physical mode, you cannot create a second migration job before the existing data migration is completed.
```
Comment (Frank945946) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:216]:
```suggestion
- When you use physical mode, you cannot create a second migration job or import task for the TiDB Cluster before the existing data migration is completed.
```

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 9]
```diff
@@ -195,10 +205,32 @@ In the **Choose the objects to be migrated** step, you can choose existing data
...
+- Physical mode is available only for TiDB clusters in AWS regions.
+
+Physical mode exports the upstream data as fast as possible, so [different specifications](/tidb-cloud/tidb-cloud-billing-dm.md#specifications-for-data-migration) have different performance impacts on QPS and TPS of the upstream database during data export. The following table shows the performance regression of each specification.
+
+| Migration specifications | Performance regression of the upstream database | Maximum Export speed |
+|-------|-------|----------|
+| 2RCU  | 15.6% | 80.84M/s |
+| 4RCU  | 20.0% | 214.2M/s |
+| 8RCU  | 28.9% | 365.5M/s |
+| 16RCU | 46.7% | 424.6M/s |
```
Comment (Frank945946) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:226]: 这里建议导出性能放在第二列，性能衰退放在第三列，更顺一些。

Comment (hfxsd) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:226]:
```suggestion
| Migration specifications |  Maximum Export speed | Performance regression of the upstream database |
|---------|-------------|--------|
| 2RCU   | 80.84 M/s  | 15.6% |
| 4RCU   | 214.2 M/s  | 20.0% |
| 8RCU   | 365.5 M/s  | 28.9% |
| 16RCU | 424.6 M/s  | 46.7% |
```

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 10]
```diff
@@ -195,10 +205,32 @@ In the **Choose the objects to be migrated** step, you can choose existing data
 
 To migrate data to TiDB Cloud once and for all, choose both **Existing data migration** and **Incremental data migration**, which ensures data consistency between the source and target databases.
 
+You can use [physical mode](https://docs.pingcap.com/tidb/stable/tidb-lightning-physical-import-mode) or [logical mode](https://docs.pingcap.com/tidb/stable/tidb-lightning-logical-import-mode) to migrate **existing data**.
+
+- The default mode is **Logical mode**. This mode supports migrating data into a table with existing data. But the performance is slower than physical mode.
+
+- It is recommended to use **Physical mode** for large datasets. When you use this mode, the target table must be empty. For the specification of 16RCU, the performance is about 2.5 times faster than logical mode. The performance of other specifications can also increase by 20% to 50% compared with logical mode.  Note that the performance data is for reference only and might vary in different scenarios.
```
Comment (hfxsd) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:212]:
```suggestion
- It is recommended to use **Physical mode** for large datasets. When you use this mode, the target table must be empty. For the specification of 16RCU, the performance is about 2.5 times faster than logical mode. The performance of other specifications can also increase by 20% to 50% compared with logical mode. Note that the performance data is for reference only and might vary in different scenarios.
```

**Review by Frank945946 — APPROVED** [review]

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 11]
```diff
@@ -195,10 +205,32 @@ In the **Choose the objects to be migrated** step, you can choose existing data
...
+- Physical mode is available only for TiDB clusters in AWS regions.
+
+Physical mode exports the upstream data as fast as possible, so [different specifications](/tidb-cloud/tidb-cloud-billing-dm.md#specifications-for-data-migration) have different performance impacts on QPS and TPS of the upstream database during data export. The following table shows the performance regression of each specification.
+
+| Migration specifications |  Maximum Export speed | Performance regression of the upstream database |
+|---------|-------------|--------|
+| 2RCU   | 80.84 MB/s  | 15.6% |
+| 4RCU   | 214.2 MB/s  | 20.0% |
+| 8RCU   | 365.5 MB/s  | 28.9% |
+| 16RCU | 424.6 MB/s  | 46.7% |
```
Comment (hfxsd) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:226]:
```suggestion
| 2RCU   | 80.84 MiB/s  | 15.6% |
| 4RCU   | 214.2 MiB/s  | 20.0% |
| 8RCU   | 365.5 MiB/s  | 28.9% |
| 16RCU | 424.6 MiB/s  | 46.7% |
```

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 12]
```diff
@@ -14,25 +14,35 @@ If you want to migrate incremental data only, see [Migrate Incremental Data from
...
 
+- When you delete a cluster in TiDB Cloud, all migration jobs in that cluster are automatically deleted and not recoverable.
+
+### Limitations of existing data migration
+
 - During existing data migration, if the table to be migrated already exists in the target database with duplicated keys, the duplicate keys will be replaced.
 
-- During incremental data migration, if the table to be migrated already exists in the target database with duplicated keys, an error is reported and the migration is interrupted. In this situation, you need to make sure whether the upstream data is accurate. If yes, click the "Restart" button of the migration job and the migration job will replace the downstream conflicting records with the upstream records.
+- If your dataset size is smaller than 1 TiB, it is recommended that you use logical mode (the default mode). If your dataset size is larger than 1 TiB, or you want to migrate existing data faster, you can use physical mode. For more information, see [Migrate existing data and incremental data](#migrate-existing-data-and-incremental-data).
```
Comment (Oreoxmt) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:39]:
```suggestion
- If your dataset size is smaller than 1 TiB, it is recommended that you use [logical mode](https://docs.pingcap.com/tidb/stable/tidb-lightning-logical-import-mode) (the default mode). If your dataset size is larger than 1 TiB, or you want to migrate existing data faster, you can use [physical mode](https://docs.pingcap.com/tidb/stable/tidb-lightning-physical-import-mode). For more information, see [Migrate existing data and incremental data](#migrate-existing-data-and-incremental-data).
```

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 13]
```diff
@@ -195,10 +205,32 @@ In the **Choose the objects to be migrated** step, you can choose existing data
...
+- Physical mode is available only for TiDB clusters in AWS regions.
+
+Physical mode exports the upstream data as fast as possible, so [different specifications](/tidb-cloud/tidb-cloud-billing-dm.md#specifications-for-data-migration) have different performance impacts on QPS and TPS of the upstream database during data export. The following table shows the performance regression of each specification.
+
+| Migration specifications |  Maximum Export speed | Performance regression of the upstream database |
+|---------|-------------|--------|
+| 2RCU   | 80.84 MiB/s  | 15.6% |
+| 4RCU   | 214.2 MiB/s  | 20.0% |
+| 8RCU   | 365.5 MiB/s  | 28.9% |
+| 16RCU | 424.6 MiB/s  | 46.7% |
```
Comment (Oreoxmt) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:226]:
```suggestion
- The default mode is **logical mode**. This mode supports migrating data into a table with existing data. But the performance is slower than physical mode.

- It is recommended to use **physical mode** for large datasets. When you use this mode, the target table must be empty. For the specification of 16 RCUs (Replication Capacity Units), the performance is about 2.5 times faster than logical mode. The performance of other specifications can increase by 20% to 50% compared with logical mode. Note that the performance data is for reference only and might vary in different scenarios.

Physical mode has the following limitations:

- When you use physical mode, you cannot create a second migration job or import task for the TiDB cluster before the existing data migration is completed.
- Physical mode is only available for TiDB clusters deployed on AWS.

Physical mode exports the upstream data as fast as possible, so [different specifications](/tidb-cloud/tidb-cloud-billing-dm.md#specifications-for-data-migration) have different performance impacts on QPS and TPS of the upstream database during data export. The following table shows the performance regression of each specification.

| Migration specification |  Maximum export speed | Performance regression of the upstream database |
|---------|-------------|--------|
| 2 RCUs   | 80.84 MiB/s  | 15.6% |
| 4 RCUs   | 214.2 MiB/s  | 20.0% |
| 8 RCUs   | 365.5 MiB/s  | 28.9% |
| 16 RCUs | 424.6 MiB/s  | 46.7% |
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**